sentry/config.example.yml

```diff
@@ -95,9 +95,12 @@ transaction-events.force-disable-internal-project: true
 #####################
 
 # Refer to https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463 for setup instructions.
+# Or https://develop.sentry.dev/integrations/slack/
 
 # slack.client-id: <'client id'>
 # slack.client-secret: <client secret>
+# slack.signing-secret: <signing secret>
+## if you made your slack bot before july 2020 set legacy-app to True
+## and use verfication-token instead of signing-secret
 # slack.verification-token: <verification token>
-## only uncomment legacy-app if you made your slack bot after july 2020
-# slack.legacy-app: False
\ No newline at end of file
+slack.legacy-app: False
```

Comment:
I'd move this above the verification token or reword the comment so the ordering matches (first token, then legacy-app)

Reply:
how about that?

Reply:
Nice :)